modules/spring-config/src/main/resources/META-INF/spring.schemas

```diff
@@ -9,11 +9,14 @@ http\://www.mulesoft.org/schema/mule/schemadoc/3.4/mule-schemadoc.xsd=META-INF/m
 http\://www.mulesoft.org/schema/mule/core/3.5/mule.xsd=META-INF/mule.xsd
 http\://www.mulesoft.org/schema/mule/schemadoc/3.5/mule-schemadoc.xsd=META-INF/mule-schemadoc.xsd
 http\://www.mulesoft.org/schema/mule/schemadoc/3.6/mule-schemadoc.xsd=META-INF/mule-schemadoc.xsd
+http\://www.mulesoft.org/schema/mule/core/3.7/mule.xsd=META-INF/mule.xsd
+http\://www.mulesoft.org/schema/mule/schemadoc/3.7/mule-schemadoc.xsd=META-INF/mule-schemadoc.xsd
 http\://www.mulesoft.org/schema/mule/core/current/mule.xsd=META-INF/mule.xsd
 http\://www.mulesoft.org/schema/mule/schemadoc/current/mule-schemadoc.xsd=META-INF/mule-schemadoc.xsd          ]
```

Comment:
good catch
it was already there but as per diff. I've updated: https://www.mulesoft.org/jira/browse/MULE-8544 to cover it and be able to backport